### ISSUE TITLE
Feature request: Refresh button

### ISSUE THREAD

**Issue by Brog33** [validatedev/istekram#5]
Since F5 doesn't do anything and there's no right click menu, it would be very nice to have a refresh button.

**Comment by validatedev** [comment]
Yeah it makes sense. I’ll look at it as soon as I’m free. Thanks.
